README.md

```diff
@@ -0,0 +1,197 @@
+# pysdccc
+
+This python packages provides a convenient way to execute the [sdccc test suite](https://github.com/Draegerwerk/sdccc/).
+
+This wrapper is only compatible with sdccc versions later than [ebe0e09](https://github.com/Draegerwerk/SDCcc/commit/ebe0e094ff92649d0bda1988b0d1c1b08403aea4).
+
+## Installation
+
+Download from pypi using `pip install pysdccc`
+
+### Development
+
+For this open source project the [Contributor License Agreement](Contributor_License_Agreement.md) governs all relevant activities and your contributions. By contributing to the project you agree to be bound by this Agreement and to licence your work accordingly.
```

Comment:
i am not sure, but i think we should use the lowercase letter here: ... bound to this **a**greement and ...

Reply:
legal stuff...